File: Format Name/nameformatting.c
#include <stdio.h>
#include <ctype.h>

int main() {
    char lastname[1000001], firstname[1000001];
    scanf("%[^,], %s", lastname, firstname);

    printf("%c. %c%s\n", toupper(firstname[0]), toupper(lastname[0]), lastname + 1);

    return 0;
}

File: Format Name/nameformatting.java
import java.util.Scanner;

public class nameformatting {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String lastname = scanner.next();
        lastname = lastname.substring(0, lastname.length() - 1);
        String firstname = scanner.next();

        System.out.println(Character.toUpperCase(firstname.charAt(0)) + ". " + Character.toUpperCase(lastname.charAt(0)) + lastname.substring(1));
    }
}
